README.md

```diff
@@ -10,13 +10,25 @@
 ## Introduction
 A go framework for Aries
 
-## Contributing
+## Build
+##### Prerequisites
+- Go 1.12
+- Docker v18.09.2
```

Comment:
This seems very specific?

Reply:
@troyronda I'll update to '>v18.09.2'.

Reply:
@troyronda on a second thought, there is not really a version requirement for Docker. Hence [removed](https://github.com/hyperledger/aries-framework-go/compare/9e512eec533a9845abfe2604f7f636c61c9284a9..77de294918da55729525425ed777814198cf215f#diff-04c6e90faac2675aa89e2176d2eec7d8R16) the version details.